src/sentry/flags/models.py

```diff
@@ -21,13 +22,16 @@ def to_string(cls, integer):
             return "deleted"
         if integer == 2:
             return "updated"
+        if integer == 3:
+            return "cloned"
         raise ValueError
 
 
 ACTION_MAP = {
     "created": ActionEnum.CREATED.value,
     "deleted": ActionEnum.DELETED.value,
     "updated": ActionEnum.UPDATED.value,
+    "cloned": ActionEnum.CLONED.value,
```

Comment:
I assume this is flag rules being cloned to a new flag name?  Maybe this could just be created instead?

Reply:
<img width="780" alt="SCR-20241016-kfki" src="https://github.com/user-attachments/assets/1f2c88fc-6975-435f-93e0-26f9efeae287">
yup, this is the description from LD. but i definitely can have it be in the "created" action bucket instead